dev/TabView/TabView.cpp

```diff
@@ -31,13 +31,15 @@ TabView::TabView()
 
     Loaded({ this, &TabView::OnLoaded });
     SizeChanged({ this, &TabView::OnSizeChanged });
+    KeyDown({ this, &TabView::OnTabViewKeyDown });
 }
```

Comment:
I think overriding OnKeyDown would be what you'd want, usually controls override that virtual instead of subscribing to the event, so the ordering is more well-defined.

Reply:
OK, makes sense. Fixed.